src/txn/txn_rollback_to_stable.c

```diff
@@ -694,9 +694,9 @@ __rollback_verify_ondisk_page(
     /* Review updates that belong to keys that are on the disk image. */
     WT_ROW_FOREACH (page, rip, i) {
         __wt_row_leaf_value_cell(session, page, rip, NULL, vpack);
-        WT_ASSERT(session, vpack->start_ts <= rollback_timestamp);
-        if (vpack->stop_ts != WT_TS_MAX)
-            WT_ASSERT(session, vpack->stop_ts <= rollback_timestamp);
+        WT_ASSERT(session, vpack->durable_start_ts <= rollback_timestamp);
+        if (vpack->durable_stop_ts != WT_TS_NONE)
```

Comment:
Not sure I understand the `MAX => NONE` change.

Reply:
All durable timestamps are initialized with WT_TS_NONE. Earlier the comparison was done with stop_ts that was initialized with WT_TS_MAX, but with the addition of durable timestamps to cell unpack, it is better to compare against them.